### ISSUE TITLE
Error when trying to import profile I made in this addon yesterday.

### ISSUE THREAD

**Issue by Larsy93** [ElvUI-TBC/ElvUI#109]
Message: Interface\AddOns\ElvUI\core\staticpopups.lua:776: attempt to call method 'SetCountInvisibleLetters' (a nil value)
Time: 06/06/17 10:41:24
Count: 3
Stack: (tail call): ?
Interface\AddOns\ElvUI\core\staticpopups.lua:776: in function `StaticPopup_Show'
Interface\AddOns\ElvUI\core\distributor.lua:424: in function <Interface\AddOns\ElvUI\core\distributor.lua:407>
Interface\AddOns\ElvUI\core\distributor.lua:467: in function `ImportProfile'
Interface\AddOns\ElvUI_Config\core.lua:300: in function <Interface\AddOns\ElvUI_Config\core.lua:295>
(tail call): ?
[C]: ?
[string "safecall Dispatcher[4]"]:9: in function <[string "safecall Dispatcher[4]"]:5>
(tail call): ?
...Ons\ElvUI_Config\Libraries\AceGUI-3.0\AceGUI-3.0.lua:314: in function `Fire'
...ies\AceGUI-3.0\widgets\AceGUIWidget-Button-ElvUI.lua:22: in function <...ies\AceGUI-3.0\widgets\AceGUIWidget-Button-ElvUI.lua:19>

**Comment by Larsy93** [comment]
Message: Interface\AddOns\ElvUI\core\staticpopups.lua:776: attempt to call method 'SetCountInvisibleLetters' (a nil value)
Time: 06/06/17 10:50:05
Count: 1
Stack: (tail call): ?
Interface\AddOns\ElvUI\core\staticpopups.lua:776: in function `StaticPopup_Show'
Interface\AddOns\ElvUI\core\distributor.lua:424: in function <Interface\AddOns\ElvUI\core\distributor.lua:407>
Interface\AddOns\ElvUI\core\distributor.lua:467: in function `ImportProfile'
Interface\AddOns\ElvUI_Config\core.lua:300: in function <Interface\AddOns\ElvUI_Config\core.lua:295>
(tail call): ?
[C]: ?
[string "safecall Dispatcher[4]"]:9: in function <[string "safecall Dispatcher[4]"]:5>
(tail call): ?
...Ons\ElvUI_Config\Libraries\AceGUI-3.0\AceGUI-3.0.lua:314: in function `Fire'
...ies\AceGUI-3.0\widgets\AceGUIWidget-Button-ElvUI.lua:22: in function <...ies\AceGUI-3.0\widgets\AceGUIWidget-Button-ElvUI.lua:19>

**Comment by Larsy93** [comment]
Accidentaly closed the issue, struggling to import this profile i made yesterday. It worked before.

**Comment by Loaal** [comment]
Issue #101 was fixed yesterday 936de60
Make sure you using last version

**Comment by Larsy93** [comment]
Iam using the latest version, let me try again.

**Comment by Larsy93** [comment]
Ok, something was wrong on my end, works with the brand new version.
